### DISCUSSION TITLE
Add Global equivalent domains for Internet Archive

### DISCUSSION THREAD

**Discussion by TheBestPessimist** [bitwarden/clients#5329] [Password Manager]
### Code Contribution Proposal

Add <kbd>Domain Rules</kbd> / <kbd>Global equivalent domains</kbd> for Internet Archive:
- `archive.org`
- `openlibrary.org`

PR: https://github.com/bitwarden/server/pull/2875

**Comment by dwbit** [comment]
Thanks for the proposal! We will review and get back to you shortly!

**Reply by TheBestPessimist** [reply]
Hey, how much does _shortly_ mean in Bitwarden time? Does it mean  _" soon™ "_ ?

**Comment by dwbit** [comment]
Hey @TheBestPessimist after internal discussions, we decided not to go forward in merging this PR. It poses a security risk for us for now as we do not know if the site(s) are safe, and we would need to validate these and all other global domains submitted, which might take up some internal capacity.  We are currently looking into ways to take this burden off the community as we look to keep these domains up-to date.  You can add the custom equivalent domains in your personally, as outlined [here](https://bitwarden.com/help/uri-match-detection/#equivalent-domains) .This PR will now be closed.

**Reply by TheBestPessimist** [reply]
@dwbit please see this reply: https://github.com/bitwarden/server/pull/2875#issuecomment-1611274356
